services/QuillLMS/lib/data/rules.csv

```diff
@@ -0,0 +1,115 @@
+﻿Overall Rule Priority,Rule,Module,Algorithm,Rule or API?,Rule Description,Status,Rule UID,Feedback - Revised,Feedback Layers,Unique feedback for each activity?,Give feedback for Turk workers?,Bolding?,Bolding: Is this what the engine returns now?,Text highlighting?,Application,Spelling corrected in background?,Reported to teachers?,Combined Concept,Notes,Notes for Feedback,Level 2 Concept,Level 1 Concept,Level 0 Concept,Feedback - Original,API Priority,Rule Priority - Opinion,Uses SpaCy?,Status Notes
+1,Pre-filter API,Pre-filter,,API,API,Live Algorithm,fe42a315-fd58-407a-b4af-d2bee277a802,,1,No,Yes,,,No,Universal: all prompts/passages,No,No,,,,,,,,1,,,
+1,Profanity,Pre-filter,Profanity,Rule,Response contains profanity from the list in the card,Live Algorithm,1e705723-50b1-438d-8b17-08d28ee895e5,[Update] Revise your work. Avoid using profanity in academic writing.,1,No,Yes,Bold the inappropriate word in the student's response,No,No,Universal: all prompts/passages,No,No,Activity Guidelines | Comprehension Activity Guidelines | Avoiding Profanity,"This is front end. It won't count as an attempt. ",Feedback should suggest that there's either a misspelling or inappropriate word,Activity Guidelines,Comprehension Activity Guidelines,Avoiding Profanity,Check your work. Then is used for time. Than is used to compare.,,,,
+2,Multiple sentences,Pre-filter,Multiple sentences,Rule,Response contains multiple sentences (period followed by a space and another character). See card for exceptions.,Live Algorithm,60a8a095-62aa-497e-84f3-e7db018afa4f,[Update] Revise your work. Make sure your response is only one sentence.,1,No,Yes,No,Yes,No,Universal: all prompts/passages,No,No,Activity Guidelines | Comprehension Activity Guidelines | Writing a Single Sentence,Front-end only for now. Convert to back-end in the future.,Feedback should suggest that there's either a punctuation error or there are multiple sentences,Activity Guidelines,Comprehension Activity Guidelines,Writing a Single Sentence,"Revise your work. Your response should be only one sentence long. ",,,,
+3,"Responses that are too short ",Pre-filter,Responses that are too short,Rule,Response is fewer than 3 words,Live Algorithm,76f519bd-ad7a-4b88-8ee6-c00a877e2ccc,Revise your work. Use the text to add more information to your response.,1,No,No,No,Yes,No,Universal: all prompts/passages,No,No,Activity Guidelines | Comprehension Activity Guidelines | Response Meets Length Requirements,"Old version: Response Too Short
+Front-end only for now. Convert to back-end in the future.",,Activity Guidelines,Comprehension Activity Guidelines,Response Meets Length Requirements,"Revise your work. When writing your response, make sure to use appropriate language.",,,,

```

Comment:
Does this line have unbalanced quotes? Is that going to cause import/parsing issues?

Reply:
All rows were parsed correctly in the latest CSV (which is fresher than this comment)